main.go

```diff
@@ -38,30 +37,28 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	reporter, err := output.GetReporter(config.Reports)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	plan := plan.New(config)
 
-	fmt.Printf("Waiting on WAIT_FOR=%v\n\n", plan.Config.WaitForHosts)
+	if err = reporter.Start(plan); err != nil {
```

Comment:
nit: use `err := ` here. where possible, you want the variable to be scoped as narrowly as possible